### ISSUE TITLE
Vex commit used to generate slim versions of the CPU

### ISSUE THREAD

**Issue by kgugala** [tcal-x/pythondata-cpu-vexriscv#4]
Generated Verilog file (https://github.com/tcal-x/pythondata-cpu-vexriscv/commit/5bb91146a62643cf606443d5366c69720906549c#diff-63516b47d7d1d76ef7a8d9ef1694bad1cec7d11cbfa4f872527bc3b4329dad12R3) points to a commit I cannot find in Vex repo. @tcal-x can you point me to the code you used?

**Comment by tcal-x** [comment]
Which hash are you talking about?  

adf552d is the commit from the spinalHDL repo.

The other hash in the generated Verilog is for this repo (pythondata-cpu-vexriscv).

From a commit here, you can find the Vex submodule commit.

But, what is the underlying goal?

**Comment by tcal-x** [comment]
I think the Cfu variants were built at https://github.com/SpinalHDL/VexRiscv/tree/21d24eb07fad16fd671b3d4668d302cbaa4549a7

There was a breaking change introduced more recently that removed a signal.   Changes in CFU Playground will be necessary to transition to current VexRiscv head.  See https://github.com/enjoy-digital/litex/issues/1036 .

**Comment by tcal-x** [comment]
> There was a breaking change introduced more recently that removed a signal. Changes in CFU Playground will be necessary to transition to current VexRiscv head. See [enjoy-digital/litex#1036](https://github.com/enjoy-digital/litex/issues/1036) .

Of course, we *do* want to make this transition, and use upstream directly.

**Comment by kgugala** [comment]
thanks. I'll check https://github.com/SpinalHDL/VexRiscv/tree/21d24eb07fad16fd671b3d4668d302cbaa4549a7
